builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -160,7 +167,7 @@ node  {
                 def serviceMetadataJson = [:]
                 if(service_config['provider'] == 'aws'){
 					def roleARN
-                    if (service_config['domain'] == "jazz") {
+                    if (service_config['domain'] && service_config['domain'] == "jazz") {
```

Comment:
service_config['domain'] is a required field now. Check not required.

Reply:
not needed. so removed it.